dapps/multisig-toolkit/src/routes/signature-analyzer.tsx

```diff
@@ -110,8 +97,32 @@ export default function SignatureAnalyzer() {
 					setError(null);
 
 					try {
-						const parsedSignature = toParsedSignaturePubkeyPair(signature);
-						setListSignaturePubkeys(parsedSignature);
+						const parsedSignature = parseSerializedSignature(signature);
+
+						if (parsedSignature.signatureScheme === 'MultiSig') {
+							const partialSignatures = parsePartialSignatures(parsedSignature.multisig);
+
+							setListSignaturePubkeys(
+								partialSignatures.map((signature) => {
+									return {
+										signatureScheme: signature.signatureScheme,
+										publicKey: signature.publicKey,
+										signature: signature.signature,
+									};
+								}),
+							);
+						} else {
+							setListSignaturePubkeys([
```

Comment:
I'm not sure if this was supposed to handle analyzing non-multisig keys, we definitely need some manual testing to make sure this all works as expected